src/PostProcess/babylon.circleOfConfusionPostProcess.ts

```diff
@@ -23,7 +23,7 @@ module BABYLON {
         /**
          * Creates a new instance of @see CircleOfConfusionPostProcess
          * @param name The name of the effect.
-         * @param scene The scene the effect belongs to.
+         * @param minMaxZCamera The camera to be used to calculate the min/max z for the depth map

```

Comment:
Now this constructor takes 2 camera parameters which is kind of weird, but the second one must be null when used in the pipeline so I think this needs to be done.

Reply:
I would prefer saving min and max as property of the depth texture as we know them when we create the depth texture

Reply:
Sounds good, changed circle of confusion and depth of field to take cameraMinZ:number, cameraMaxZ:number, depthTexture: RenderTargetTexture as parameters instead